txmatching/utils/hla_system/hla_crossmatch.py

```diff
@@ -111,6 +114,15 @@ def from_crossmatched_antibodies(cls, assumed_hla_types: List[HLATypeWithFrequen
         antibody_matches = cls._find_common_matches(assumed_hla_types, crossmatched_antibodies)
         return cls(assumed_hla_types, antibody_matches, all_antibodies)
 
+    def check_if_positive_crossmatch(self):
```

Comment:
```suggestion
    def is_positive_crossmatch(self):
```

Jinak lze chápat jako "udělat nějaký check, pokud je positivní crossmatch"